src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,48 @@
+package duke.ui;
+
+import java.util.Scanner;
+
+public class Ui {
```

Comment:
This class has very well-defined methods, which makes the rest of the code in the other classes very easy to read!! However, I think that the purpose of the printIndented() function could be abit confusing for those reading the code, so perhaps that could be better explained or named!